packages/web-app-files/src/index.ts

```diff
@@ -126,7 +126,7 @@ export const navItems = (context): AppNavigationItem[] => {
 }
 
 export default defineWebApplication({
-  setup(args) {
+  setup() {
```

Comment:
Was the linter really complaining about `args` here? Because we passed it down to `extensions()` down below 🤔

Reply:
Removed them, because the extension function doesn't do anything with it. I'm not sure if this is something that should be available so it can be modified by other developers and just give it exception with disabling eslint.

Reply:
Ahh okay, now I get it. It's okay then I'd say. If the `extensions` method needs it one day we can re-add it.